saleor/payment/gateways/adyen/tests/webhooks/test_handle_notifications.py

```diff
@@ -480,6 +480,7 @@ def test_handle_capture_with_payment_already_charged(
         type=OrderEvents.EXTERNAL_SERVICE_NOTIFICATION
     )
     assert external_events.count() == 1
+    assert payment.captured_amount == payment.total
```

Comment:
Question to the core team: I broke this test by accident, is it intended behaviour to overcharge here? It might be relevant for making the decision whether overpaid exists do really exist

Reply:
@korycins What's the action point here? Do we move it to a separate task?

Reply:
Transaction with kind.Capture is missing in the test.